static/js/server_events_dispatch.js

```diff
@@ -22,6 +22,13 @@ exports.dispatch_normal_event = function dispatch_normal_event(event) {
             event.hotspots;
         break;
 
+    case 'invites_changed':
+        if ($('#admin-invites-status').length) {
+            ui_report.success(i18n.t("Invites changed. You need to reload the window."),
```

Comment:
Maybe 
"Updates available. Refresh to see changes."
with the whole think a link you can click, that refreshes the page?